test/e2e/lib/pubsub.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lib
+
+import (
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/google/knative-gcp/pkg/apis/events/v1alpha1"
+	kngcptesting "github.com/google/knative-gcp/pkg/reconciler/testing"
+	"github.com/google/knative-gcp/test/e2e/lib/metrics"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pkgmetrics "knative.dev/pkg/metrics"
+)
+
+func MakePubSubOrDie(t *testing.T, client *Client, gvk metav1.GroupVersionKind, psName, targetName, topicName string, so ...kngcptesting.CloudPubSubSourceOption) {
```

Comment:
This line is a bit long, add line breaks?

Reply:
done